config/postgres/docker-entrypoint-initdb.d/20-create-app-db.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+psql -v ON_ERROR_STOP=1 \
+    --dbname "$POSTGRES_DB" <<-EOSQL
+    CREATE DATABASE ${APP_DB} OWNER ${APP_USER};
```

Comment:
Should a test base be created here?

Reply:
Currently when you use command `bin/phing test then phing executes tasks 
`database:drop, database:create`
Even if i add create db for test then `phing` will remove this db and create..